src/components/Popover/Popover.js

```diff
@@ -23,18 +21,18 @@ export class Popover extends React.Component {
   }
 
   renderHeader() {
-    const { header, panels, customLayout } = this.props;
+    const { header, panels, customHeaderTheme } = this.props;
     const headerClasses = [
       'slds-popover__header',
-      { 'slds-theme_warning': customLayout === 'warning' },
-      { 'slds-theme_error': customLayout === 'error' },
-      { 'slds-theme_success': customLayout === 'success' },
-      { 'slds-theme_info': customLayout === 'info' },
+      { 'slds-theme_warning': customHeaderTheme === 'warning' },
+      { 'slds-theme_error': customHeaderTheme === 'error' },
+      { 'slds-theme_success': customHeaderTheme === 'success' },
+      { 'slds-theme_info': customHeaderTheme === 'info' },
     ];
 
     let headerContent;
     let borderRadius;
-    if (!!panels && (typeof customLayout !== 'undefined') && customLayout !== '') {
+    if (!!panels && (typeof customHeaderTheme !== 'undefined') && customHeaderTheme !== '') {
```

Comment:
I know it's not your code here, but checking for explicit undefined (especially via typeof) is bad practice.